scp/lib/timetable/theorySection.dart

```diff
@@ -179,6 +179,7 @@ Widget _buildCarousel(_TheorySectionState timetableState, BuildContext context,
                 page = pageController.page;
               });
             }
+            return false;

```

Comment:
@nayakastha Why this addition?

Reply:
`onNotification` expects a bool and previously the function was returning a void. So it didn't make sense